ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/advanced-gas-fee-inputs-basefee.js

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
+import {
+  divideCurrencies,
+  multiplyCurrencies,
+} from '../../../../../shared/modules/conversion.utils';
+import { PRIMARY, SECONDARY } from '../../../../helpers/constants/common';
+import {
+  decGWEIToHexWEI,
+  hexWEIToDecGWEI,
+} from '../../../../helpers/utils/conversions.util';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { useUserPreferencedCurrency } from '../../../../hooks/useUserPreferencedCurrency';
+import { useCurrencyDisplay } from '../../../../hooks/useCurrencyDisplay';
+import Button from '../../../ui/button';
+import FormField from '../../../ui/form-field';
+import I18nValue from '../../../ui/i18n-value';
+
+import AdvancedGasFeeInputSubtext from '../advanced-gas-fee-input-subtext';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+
+const devideCurrencyValues = (value, baseFee, numberOfDecimals) => {
+  if (baseFee > 1)
```

Comment:
Can we add braces if the `if` statement falls across multiple lines?

Reply:
done